SAMv1.tex

```diff
@@ -539,12 +540,66 @@ \subsection{The alignment section: mandatory fields}\label{sec:alnrecord}
   the next read. If {\sf PNEXT} is 0, no assumptions can be made on
   {\sf RNEXT} and bit 0x20.
 \item {\sf TLEN}: signed observed Template LENgth. If all segments are
-  mapped to the same reference, the unsigned observed template length
-  equals the number of bases from the leftmost mapped base to the
-  rightmost mapped base. The leftmost segment has a plus sign and the
-  rightmost has a minus sign. The sign of segments in the middle is
-  undefined. It is set as 0 for single-segment template or when the
-  information is unavailable.
+  mapped to the same reference sequence, the unsigned observed template length
```

Comment:
I would prefer  " the absolute value of TLEN equals...." otherwise is sounds like there's a "signed observed template length" and also a "signed observed template length"...